src/api/index.tsx

```diff
@@ -76,6 +76,24 @@ export const callApi = async ({
   }
 };
 
+type GetVersionFileParams = {
+  apiState: ApiState;
+  filename?: string;
+  versionId: number;
+};
+
+export const getVersionFile = async ({
+  apiState,
+  filename = '',
```

Comment:
why is there a default empty string value?

Reply:
Not sure. I've removed it.